SConstruct

```diff
@@ -60,6 +59,9 @@ opts.Add(
     )
 )
 
+opts.Add(BoolVariable("cache", "Cache results for faster iteration when develloping.", False))
+opts.Add(PathVariable("cache_dir", "Where you want your cache directory.", "cache", PathVariable.PathIsDirCreate))
```

Comment:
For consistency with the godot build system we should probably use an env variable instead of options.
I,e,:

```
scons_cache_path = os.environ.get("SCONS_CACHE")
if scons_cache_path is not None:
   CacheDir(scons_cache_path)
   ....
```

Reply:
I.e. remove this completely:
```suggestion
```

Reply:
I will have a look at Godot's and do something like this.
Does this mean that caching could only be activated through an environment variable and not through CLI options at all?
I see no harm if this is required, but I often find it very convenient just to add an option.

Reply:
It's still kinda the same, just add the "option" before scons:
```
SCONS_CACHE=/path/to/cache scons OPTIONS
```

or, add it to your env variable I guess.

Reply:
Yes I simply prefer the option way, but no problem I can deal with it 🙂